File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/GSRLocation.java
package com.pennapps.labs.pennmobile.classes;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by MikeD on 1/27/2018.
 */

public class GSRLocation {

    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("id")
    @Expose
    public Integer id;
    @SerializedName("service")
    @Expose
    public String service;
}

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/GSRLocation.kt
package com.pennapps.labs.pennmobile.classes

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Varun on 10/14/2018.
 */

class GSRLocation {

    @SerializedName("name")
    @Expose
    @JvmField var name: String? = null

    @SerializedName("lid")
    @Expose
    @JvmField var id: Int = 0

    @SerializedName("gid")
    @Expose
    @JvmField var gid: Int = 0

    @SerializedName("kind")
    @Expose
    @JvmField var kind: String? = null


}